### PR TITLE
Migrate PSV Linux Clang build Travis>Azure

### PR DESCRIPTION
**Pull request by ystefinko**
Azure is much more stable platform. Will decrease queues
on Travis in peak time. Travis image update from old
xenial to new bionic. Travis claims some faster image
loading and improvements in bionic. Also removed not
needed packages. Fix 2 minor errors in geo/tiling and Auth.

Signed-off-by: Yaroslav Stefinko <ext-yaroslav.stefinko@here.com>

### CODE REVIEW

**Comment by ystefinko** [comment]
> Please don't forget to adapt README.md also.

Done.

File: olp-cpp-sdk-authentication/src/AuthenticationClient.cpp [hunk 1]
```diff
@@ -591,19 +591,13 @@ client::CancellationToken AuthenticationClient::Impl::HandleUserRequest(
...
-          default:
-            callback(response);
-            break;
+        if (response_status == http::HttpStatusCode::OK) {
+          cache->locked([credentials, &response](
+                            utils::LruCache<std::string, SignInUserResult>& c) {
+            return c.InsertOrAssign(credentials.GetKey(), response);
+          });
         }
+        callback(response);
```
Comment (r0busta) [inline olp-cpp-sdk-authentication/src/AuthenticationClient.cpp]: This change is unrelated to the commit

Comment (ystefinko) [reply olp-cpp-sdk-authentication/src/AuthenticationClient.cpp]: It was needed to avoid error on Linux Clang build.

Comment (r0busta) [reply olp-cpp-sdk-authentication/src/AuthenticationClient.cpp]: Could you please submit it as a separate PR? So in the case, we need to revert anything, the fix stays in the master

Comment (ystefinko) [reply olp-cpp-sdk-authentication/src/AuthenticationClient.cpp]: Sorry, I mean Gcc build fixes, will fix them in separate PR.
